mainFunctionUpdates.py

```diff
@@ -14,94 +14,111 @@
                 meta_tags=[{'name': 'viewport', 
                           'content': 'width=device-width, initial-scale=1.0'}]
                )
+initialDays=7
```

Comment:
I would suggest being consistent with the style for variable naming.

I would suggest either camelCase or snake_case across the app. :)

In this case, I see snake case everywhere else so I would propose changing to snake here.